routes/send-email.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const router = express.Router()
+const fs = require('fs')
+const path = require('path')
+const ejs = require('ejs')
+const nodeMailer = require('nodemailer')
+
+router.post('/', function (req, res, next) {
+  let transporter = nodeMailer.createTransport({
+    service: 'Gmail',
+    auth: {
+      user: 'weather.rate@gmail.com',
+      pass: '1234qwerasdfFFFF'
```

Comment:
that is not right. Please move it into separate .dist file and change the gmail account password